skimage/measure/__init__.py

```diff
@@ -31,7 +36,7 @@
            'moments',
            'moments_central',
            'moments_coords',
-           'moments_coords_central'
```

Comment:
Whoa! That is, actually, a bug, and the fix has to be backported.

Reply:
should this be done via a different request?

Reply:
!!!! wow! Good catch @seanbudd (and @soupault, I missed this when reading the diff!). Yes, please submit a different PR with just this fix.

Reply:
@jni oops! Was definitely not trying to throw anyone under the bus!

Reply:
:+1: 

`git blame` is an unfortunate name for the command. Humorous at times, but on the whole a negative imho!